src/bridge/error/error_strings_test.cc

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <algorithm>
+
+#include <openssl/err.h>
+#include <gtest/gtest.h>
+
+#include "src/backing/status/status.h"
+#include "src/bridge/error/error_strings.h"
+
+namespace kmsengine {
+namespace bridge {
+namespace error_impl {
+namespace {
+
+const StatusCode kStatusCodes[] = {
+  StatusCode::kOk,
+  StatusCode::kCancelled,
+  StatusCode::kUnknown,
+  StatusCode::kInvalidArgument,
+  StatusCode::kDeadlineExceeded,
+  StatusCode::kNotFound,
+  StatusCode::kAlreadyExists,
+  StatusCode::kPermissionDenied,
+  StatusCode::kUnauthenticated,
+  StatusCode::kResourceExhausted,
+  StatusCode::kFailedPrecondition,
+  StatusCode::kAborted,
+  StatusCode::kOutOfRange,
+  StatusCode::kUnimplemented,
+  StatusCode::kInternal,
+  StatusCode::kUnavailable,
+  StatusCode::kDataLoss,
+};
+
+TEST(ErrorStringsTest, PackReasonCode) {
```

Comment:
You probably don't need to test all status codes here in order to see that `PackReasonCode` is working.